cypress/support/commands.js

```diff
@@ -47,6 +48,37 @@ const testUser = {
   UserPoolId: USER_POOL_ID,
 };
 
+Cypress.Commands.add('connectAuthor', () => {
+  cy.get('[data-testid=create-author-button]').click();
+  cy.get('[data-testid=modal_next]').click();
+});
+
+Cypress.Commands.add('createAuthority', (newAuthority, IdToken) => {
+  return new Cypress.Promise((resolve, reject) => {
+    const authority = createAuthority(newAuthority.firstName, newAuthority.lastName, newAuthority.feideid, IdToken);
```

Comment:
man kan opprette authority i appen også, så jeg tror kanskje jeg heller vill gjort det. Blir mer koding såklart, men det øker bare testdekning, så slipper vi å ha api-kallene både her og i frontend også.

Reply:
Jeg er uenig. Når en feature har som forutsetning at autoriteten finnes fra før så får man ikke testet det ved å opprette autoriteten gjennom appen.

Reply:
Vi skal sikkert ha en test på at man får opprettet autoritet også, så hvis vi kjører den testen først vet vi at den autoriteten finnes, hvis ikke er det en avdekt feil. Ender opp med at noen tester blir avhengig av rekkefølge da, men bare i de tilfellene appen også krever at brukeren gjør ting i riktig rekkefølge

Reply:
Du har et poeng med at det blir en liten mismatch med feature da, men jeg syns det gir mer mening

Reply:
Vil ikke lage tester som er avhengig av at andre tester er kjørt.

Reply:
Husk også at autoritetsregisteret er uavhengig av vår app.

Reply:
Enda et argument: siden jeg ikke kan slette autoriteter så må jeg kunne manipulere dem for å ikke fylle opp autoritetsregisteret med test-autoriteter. Noen av de tingene jeg trenger å gjøre er ikke mulig å gjøre gjennom appen.

Reply:
Ja det er sant, for opprydding må vi sikkert gjøre det slik :/
men syns det virker rart å duplisere kode fra frontend for å oppfylle forutsetninger vi også kunne fått til med funksjonalitet som er i appen.
Mulig @brinxmat eller @sarahse har noen synspunkter også?

Reply:
Det er flere ting som ikke kan gjøres gjennom appen for oppsett og opprydding. Tenker jeg merger denne så kan vi ta en diskusjon etter ferien.